src/parsers/form.ts

```diff
@@ -9,7 +9,8 @@
 
 import raw from 'raw-body'
 import inflate from 'inflation'
-import qs, { IParseOptions } from 'qs'
+import * as qs from 'neoqs'
```

Comment:
Just in case bodyparser still ships with CJS, this should be neoqs/legacy

Reply:
with v6 of adonis they migrated entirely to ESM!

Thanks for your work on neoqs and appreciate you getting involved here :)

Reply:
Great!! Thanks a lot for this PR

Heads up, I'm working on `neoqs/modern`, which will cut the bundle size from 3.9kb to 3.4kb, by removing all the option invalidation logic, which can be done by typescript entirely. I will then add a PR to switch to that, dropping the size even more

Reply:
Can we actually get away with just URLSearchParams or [node:querystring](https://nodejs.org/api/querystring.html) ?

Reply:
depending on the engines that adonis aims to support, I think that's an option as well!

Reply:
> Can we actually get away with just URLSearchParams or [node:querystring](https://nodejs.org/api/querystring.html) ?

No, we cannot, `qs` and `neoqs` supports nested object syntax, whereas the `node:querystring` modules do not.